components/sysutils/minicom/.gitignore

```diff
@@ -0,0 +1 @@
+/minicom-*/
```

Comment:
Is this really necessay? We usually only have a .gitignore in the root folder.

Reply:
It would be really great to be able to use `git status` without having to disable showing untracked files.  I think putting a `.gitignore` in each component directory that tracks the locally expected detritus makes a lot of sense.  In particular, if we move or rename a component directory, or just delete a component in future, it will be impossible to forget to update the top-level `.gitignore` as it will move or be removed with the component.

Reply:
Ok, I am convinced ;-)